src/ch/imedias/rsccfx/view/RsccSupportPresenter.java

```diff
@@ -73,8 +66,10 @@ public void initSize(Scene scene) {
   public String validationImage(String token) {
 
     if (validateToken(token)) {
+      view.connectBtn.setDisable(false);
```

Comment:
but one is a StringProperty and th other is Boolean, and the method uses a regex. I don't know how to connect all this in a binding. Therefor i leave it like this, please open an issue or apply it by yourself. Thanks